### PR TITLE
Azure functions documentation changes

### PR DESCRIPTION
**Pull request by ankurch627**
## Purpose of this pull request

This pull request edits Azure Functions app documentation

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

**Comment by jpipkin1** [comment]
@ankurch627, I converted the PR to a draft since the PR is titled "Draft". When the PR is ready for review, click the **Ready for review** button at the bottom of the page, and remove "Draft" from the PR title.

File: docs/integrations/microsoft-azure/azure-functions.md [hunk 1]
```diff
@@ -1,20 +1,25 @@
...
+<img src={useBaseUrl('img/integrations/microsoft-azure/azure-functions.png')} alt="thumbnail icon" width="75"/>
 
 [Azure Functions](https://learn.microsoft.com/en-us/azure/azure-functions/functions-overview) is a serverless solution that allows you to write less code, maintain less infrastructure, and save on costs. This integration helps in monitoring the health, tracking executions, and estimating cost of your function apps.
 
 ## Log and metric types
 
-For Azure Functions, you can collect the following logs and metrics:
+The Sumo Logic app for Azure Functions app uses the following log types:
+
+* FunctionApplicationLogs
```
Comment (JV0812) [inline docs/integrations/microsoft-azure/azure-functions.md:18]:
```suggestion
* `FunctionApplicationLogs`
```

File: docs/integrations/microsoft-azure/azure-functions.md [hunk 2]
```diff
@@ -1,20 +1,25 @@
...
+* AppServiceAuthenticationLogs
```
Comment (JV0812) [inline docs/integrations/microsoft-azure/azure-functions.md:19]:
```suggestion
* `AppServiceAuthenticationLogs`
```

File: docs/integrations/microsoft-azure/azure-functions.md [hunk 3]
```diff
@@ -1,20 +1,25 @@
...
+* AllMetrics
```
Comment (JV0812) [inline docs/integrations/microsoft-azure/azure-functions.md:20]:
```suggestion
* `AllMetrics`
```

File: docs/integrations/microsoft-azure/azure-functions.md [hunk 4]
```diff
@@ -34,6 +45,83 @@ You must explicitly enable diagnostic settings for each Azure Functions you want
...
+1. <!--Kanso [**Classic UI**](/docs/get-started/sumo-logic-ui/). Kanso--> In the main Sumo Logic menu, select **Manage Data > Logs > Fields**. <!--Kanso <br/>[**New UI**](/docs/get-started/sumo-logic-ui-new/). In the top menu select **Configuration**, and then under **Logs** select **Fields**. You can also click the **Go To...** menu at the top of the screen and select **Fields**. Kanso-->
+2. Search for following fields:
+   - `tenant_name`. This field is tagged at the collector level and you can get the tenant name using the instructions [here](https://learn.microsoft.com/en-us/azure/active-directory-b2c/tenant-management-read-tenant-name#get-your-tenant-name).
+   - `location`. The region to which the resource name belongs to.
+   - `subscription_id`. Id associated with a subscription where resource is present.
+   - `resource_group`. The resource group name where the Azure resource is present.
+   - `provider_name`. Azure resource provider name (for example, Microsoft.WEB).
+   - `resource_type`. Azure resource type (for example, SITES).
+   - `resource_name`. The name of the resource (for example, Azure Function App name).
+
```
Comment (JV0812) [inline docs/integrations/microsoft-azure/azure-functions.md:58]:
```suggestion
```

File: docs/integrations/microsoft-azure/azure-functions.md [hunk 5]
```diff
@@ -34,6 +45,83 @@ You must explicitly enable diagnostic settings for each Azure Functions you want
...
+3. Create the fields if it is not present. Refer to [create and manage fields](/docs/manage/fields/#manage-fields).
+
+### Configure Field Extraction Rules
+
+Create a Field Extraction Rule (FER) by following the instructions [here](/docs/manage/field-extractions/create-field-extraction-rule/). If the FER already exists with same name, then skip this step.
+
+* **Azure Location Extraction FER**
```
Comment (JV0812) [inline docs/integrations/microsoft-azure/azure-functions.md:65]:
```suggestion
```

File: docs/integrations/microsoft-azure/azure-functions.md [hunk 6]
```diff
@@ -47,16 +135,188 @@ In this section, you will configure a pipeline for shipping metrics from Azure M
...
 
-1. To set up the Azure Event Hubs source in Sumo Logic, refer to [Azure Event Hubs Source for Logs](/docs/send-data/collect-from-other-data-sources/azure-monitoring/ms-azure-event-hubs-source/).
-2. To create the Diagnostic settings in Azure portal, refer to the [Azure documentation](https://learn.microsoft.com/en-us/azure/azure-functions/functions-monitor-log-analytics?tabs=csharp#setting-up). Perform below steps for each Azure Functions that you want to monitor.
+#### Diagnostic logs
+
+In this section, you will configure a pipeline for shipping diagnostic logs from [Azure Monitor](https://docs.microsoft.com/en-us/azure/monitoring-and-diagnostics/monitoring-get-started) to an Event Hub.
+
+1. To set up the Azure Event Hubs source in Sumo Logic, refer to the [Azure Event Hubs Source for Logs](/docs/send-data/collect-from-other-data-sources/azure-monitoring/ms-azure-event-hubs-source/).
+1. To create the Diagnostic settings in Azure portal, refer to the [Azure documentation](https://learn.microsoft.com/en-gb/azure/data-factory/monitor-configure-diagnostics). Perform below steps for each Azure Functions that you want to monitor.
```
Comment (JV0812) [inline docs/integrations/microsoft-azure/azure-functions.md:144]:
```suggestion
1. To create the **Diagnostic setting** in the Azure portal, refer to the [Azure documentation](https://learn.microsoft.com/en-gb/azure/data-factory/monitor-configure-diagnostics). Perform the below steps for each Azure Functions that you want to monitor.
```

File: docs/integrations/microsoft-azure/azure-functions.md [hunk 7]
```diff
@@ -47,16 +135,188 @@ In this section, you will configure a pipeline for shipping metrics from Azure M
...
    * Choose `Stream to an event hub` as the destination.
```
Comment (JV0812) [inline docs/integrations/microsoft-azure/azure-functions.md:145]: Please add numbered bulletins as these are a series of steps which needs to be performed.

File: docs/integrations/microsoft-azure/azure-functions.md [hunk 8]
```diff
@@ -47,16 +135,188 @@ In this section, you will configure a pipeline for shipping metrics from Azure M
...
-   * Select `Function Application Logs`.
+   * Select `AllMetrics`.
    * Use the Event hub namespace and Event hub name configured in previous step in destination details section. You can use the default policy `RootManageSharedAccessKey` as the policy name.
 
-## Troubleshooting
+   <img src={useBaseUrl('img/integrations/microsoft-azure/Azure-Functions-Configure-Diagnostic-Metrics.png')} alt="Azure Storage Tag Location" style={{border: '1px solid gray'}} width="800" />
+
+1. Tag the location field in the source with right location value.<br/><img src={useBaseUrl('img/integrations/microsoft-azure/Azure-Storage-Tag-Location.png')} alt="Azure Storage Tag Location" style={{border: '1px solid gray'}} width="400" />
+
+#### Activity Logs
```
Comment (JV0812) [inline docs/integrations/microsoft-azure/azure-functions.md:153]:
```suggestion
#### Activity Logs (Optional)
```

File: docs/integrations/microsoft-azure/azure-functions.md [hunk 9]
```diff
@@ -47,16 +135,188 @@ In this section, you will configure a pipeline for shipping metrics from Azure M
...
+
+To collect activity logs, follow the instructions [here](/docs/integrations/microsoft-azure/audit). If you are already collecting activity logs for a subscription, do not perform this step.
+
+:::note
+Since this source contains logs from multiple regions, make sure that you do not tag this source with the location tag.
+:::
+
+##### Enabling Microsoft Defender for Cloud
+
+:::note
```
Comment (JV0812) [inline docs/integrations/microsoft-azure/azure-functions.md:163]: No need to add this as a part of note as this is the only availbale content under this section.

File: docs/integrations/microsoft-azure/azure-functions.md [hunk 10]
```diff
@@ -47,16 +135,188 @@ In this section, you will configure a pipeline for shipping metrics from Azure M
...
+For Security events, make sure you enable [Microsoft Defender for Cloud](https://learn.microsoft.com/en-us/azure/azure-sql/database/azure-defender-for-sql?view=azuresql#enable-microsoft-defender-for-sql). If you have an existing settings, click **Edit Settings**.
+:::
+
+<img src={useBaseUrl('img/integrations/microsoft-azure/Microsoft-Cloud-Defender-Edit-Settings.png')} alt="Edit Settings" style={{border: '1px solid gray'}} width="800" />
+
+<img src={useBaseUrl('img/integrations/microsoft-azure/Microsoft-Cloud-Defender-Plans-AppService.png')} alt="Cloud Defender Plans" style={{border: '1px solid gray'}} width="800" />
+
+##### Enabling Health Check Metric
+
+:::note
```
Comment (JV0812) [inline docs/integrations/microsoft-azure/azure-functions.md:173]: No need to add this as a part of note as this is the only availbale content under this section.

File: docs/integrations/microsoft-azure/azure-functions.md [hunk 11]
```diff
@@ -47,16 +135,188 @@ In this section, you will configure a pipeline for shipping metrics from Azure M
...
+For getting health check metric, make sure you enable health check by going to Monitoring tab.
```
Comment (JV0812) [inline docs/integrations/microsoft-azure/azure-functions.md:174]:
```suggestion
For getting health check metric, make sure you enable health check in the Monitoring tab.
```

File: docs/integrations/microsoft-azure/azure-functions.md [hunk 12]
```diff
@@ -47,16 +135,188 @@ In this section, you will configure a pipeline for shipping metrics from Azure M
...
+
+## Viewing the Azure Functions dashboards
+
+import ViewDashboards from '../../reuse/apps/view-dashboards.md';
+
+<ViewDashboards/>
+
+### Overview
+
+The **Azure Functions - Overview** dashboard provides information of any service health incidents or resource health events associated with Azure Functions in your azure account.
```
Comment (JV0812) [inline docs/integrations/microsoft-azure/azure-functions.md:208]:
```suggestion
The **Azure Functions - Overview** dashboard provides a comprehensive information of all the service health incidents or resource health events associated with Azure Functions in your azure account.
```

File: docs/integrations/microsoft-azure/azure-functions.md [hunk 13]
```diff
@@ -47,16 +135,188 @@ In this section, you will configure a pipeline for shipping metrics from Azure M
...
+    * View recent resource and service health incidents.
+    * View distribution of service and resource health by incident type.
+
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Azure-SQL/Azure-SQL-Health.png')} alt="Azure SQL health dashboard" style={{border: '1px solid gray'}} width="800" />
+
+
+### Cost
+
+The **Azure Functions - Cost** dashboard provides information on the expenses associated with your Azure Functions. This includes details on the cost of resources, usage trends, and cost management insights.
```
Comment (JV0812) [inline docs/integrations/microsoft-azure/azure-functions.md:220]:
```suggestion
The **Azure Functions - Cost** dashboard provides information about the expenses associated with your Azure Functions. This includes details on the cost of resources, usage trends, and cost management insights.
```

File: docs/integrations/microsoft-azure/azure-functions.md [hunk 14]
```diff
@@ -47,16 +135,176 @@ In this section, you will configure a pipeline for shipping metrics from Azure M
...
+
+Use this dashboard to:
+* Monitor and analyze your spending on Azure Functions.
+* Review cost trends and identify areas where you can optimize spending.
+* Access detailed billing information and cost breakdowns.
+
```
Comment (JV0812) [inline docs/integrations/microsoft-azure/azure-functions.md:211]: Don't we have dashboard images for rest of the dashboards?

**Review by JV0812 — CHANGES_REQUESTED** [review]
Minor formatting and dashboard images missing.

**Comment by jpipkin1** [comment]
@ankurch627, I placed the **do-not-merge** label on this PR to indicate that it should not be published yet. When the PR is ready to publish, remove the **do-not-merge** label.

File: docs/integrations/microsoft-azure/azure-functions.md [hunk 15]
```diff
@@ -1,20 +1,25 @@
...
 title: Azure Functions
-description: Learn about the Sumo Logic collection process for the Azure Functions service.
+sidebar_label: Azure Functions
+description: The Sumo Logic app for Azure Functions helps you monitor activity in Azure Functions.
 ---
 
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
-<img src={useBaseUrl('img/integrations/microsoft-azure/azure-functions.png')} alt="Thumbnail icon" width="50"/>
+<img src={useBaseUrl('img/integrations/microsoft-azure/azure-functions.png')} alt="thumbnail icon" width="75"/>
```
Comment (kimsauce) [inline docs/integrations/microsoft-azure/azure-functions.md:10]: 75px is pretty massive -- the large size pushes down the written content

```suggestion
<img src={useBaseUrl('img/integrations/microsoft-azure/azure-functions.png')} alt="thumbnail icon" width="45"/>
```

**Review by jpipkin1 — CHANGES_REQUESTED** [review]
@ankurch627, I made some minor changes (see commit "[Updates from review](https://github.com/SumoLogic/sumologic-documentation/pull/4452/commits/98acadb8d2fb715af3f7dd9da2cbe99ca3e93f77)"). 

As @JV0812 said, you need to add screen images for the rest of the dashboards.

File: docs/integrations/microsoft-azure/azure-functions.md [hunk 16]
```diff
@@ -1,27 +1,38 @@
 ---
 id: azure-functions
 title: Azure Functions
-description: Learn about the Sumo Logic collection process for the Azure Functions service.
+sidebar_label: Azure Functions
+description: The Sumo Logic app for Azure Functions helps you monitor activity in Azure Functions.
```
Comment (kimsauce) [inline docs/integrations/microsoft-azure/azure-functions.md:5]:
```suggestion
description: The Sumo Logic app for Azure Functions enables seamless integration and monitoring of Azure Functions, providing insights and analytics to optimize serverless application performance and security.
```

File: docs/integrations/microsoft-azure/azure-functions.md (hunk 14 above)
Comment (himanshu219) [reply docs/integrations/microsoft-azure/azure-functions.md:204]: yes we do have, @ankurch627  will update the urls

File: docs/integrations/microsoft-azure/azure-functions.md [hunk 17]
```diff
@@ -30,33 +41,277 @@ Azure service sends monitoring data to Azure Monitor, which can then [stream dat
...
+
+<img src={useBaseUrl('img/integrations/microsoft-azure/Microsoft-Cloud-Defender-Edit-Settings.png')} alt="Edit Settings" style={{border: '1px solid gray'}} width="800" />
+
+<img src={useBaseUrl('img/integrations/microsoft-azure/Microsoft-Cloud-Defender-Plans-AppService.png')} alt="Cloud Defender Plans" style={{border: '1px solid gray'}} width="800" />
+
+##### Enabling health check metric
+
+For getting health check metric, make sure you enable health check in the Monitoring tab.
+
+<img src={useBaseUrl('img/integrations/microsoft-azure/Enable-Health-Check-Metric.png')} alt="Cloud Defender Plans" style={{border: '1px solid gray'}} width="800" />
```
Comment (himanshu219) [inline docs/integrations/microsoft-azure/azure-functions.md:170]: update alt text to enable health check metric

File: docs/integrations/microsoft-azure/azure-functions.md [hunk 18]
```diff
@@ -30,33 +41,277 @@ Azure service sends monitoring data to Azure Monitor, which can then [stream dat
...
+
+To collect activity logs, follow the instructions [here](/docs/integrations/microsoft-azure/audit). If you are already collecting activity logs for a subscription, do not perform this step.
+
+:::note
+Since this source contains logs from multiple regions, make sure that you do not tag this source with the location tag.
+:::
+
+##### Enabling Microsoft Defender for Cloud
+
+For Security events, make sure you enable [Microsoft Defender for Cloud](https://learn.microsoft.com/en-us/azure/azure-sql/database/azure-defender-for-sql?view=azuresql#enable-microsoft-defender-for-sql). If you have an existing settings, click **Edit Settings**.
```
Comment (himanshu219) [inline docs/integrations/microsoft-azure/azure-functions.md:160]: update the link, the above link is for azure sql

File: docs/integrations/microsoft-azure/azure-functions.md (hunk 17 above)
Comment (ankurch627) [reply docs/integrations/microsoft-azure/azure-functions.md]: Done @himanshu219

File: docs/integrations/microsoft-azure/azure-functions.md [hunk 19]
```diff
@@ -30,33 +41,276 @@ Azure service sends monitoring data to Azure Monitor, which can then [stream dat
...
+
+To collect activity logs, follow the instructions [here](/docs/integrations/microsoft-azure/audit). If you are already collecting activity logs for a subscription, do not perform this step.
+
+:::note
+Since this source contains logs from multiple regions, make sure that you do not tag this source with the location tag.
+:::
+
+##### Enabling Microsoft Defender for Cloud
+
+For Security events, make sure you enable [Microsoft Defender for Cloud](https://learn.microsoft.com/en-us/azure/defender-for-cloud/tutorial-enable-app-service-plan#enable-the-defender-for-app-service-plan). If you have an existing settings, click **Edit Settings**.
```
Comment (JV0812) [inline docs/integrations/microsoft-azure/azure-functions.md:159]: Can you be bit more elaborate. The steps/UI in the screenshots do not match easily with the content here.

File: docs/integrations/microsoft-azure/azure-functions.md [hunk 20]
```diff
@@ -30,33 +41,276 @@ Azure service sends monitoring data to Azure Monitor, which can then [stream dat
...
+
+<img src={useBaseUrl('img/integrations/microsoft-azure/Microsoft-Cloud-Defender-Edit-Settings.png')} alt="Edit Settings" style={{border: '1px solid gray'}} width="800" />
+
+<img src={useBaseUrl('img/integrations/microsoft-azure/Microsoft-Cloud-Defender-Plans-AppService.png')} alt="Cloud Defender Plans" style={{border: '1px solid gray'}} width="800" />
+
+##### Enabling health check metric
+
+For getting health check metric, make sure you enable health check in the Monitoring tab.
```
Comment (JV0812) [inline docs/integrations/microsoft-azure/azure-functions.md:167]: Add a red colored box to the focus point in the page.

File: docs/integrations/microsoft-azure/azure-functions.md [hunk 21]
```diff
@@ -30,33 +41,276 @@ Azure service sends monitoring data to Azure Monitor, which can then [stream dat
...
+### Network
+
+The **Azure Functions - Network** dashboard provides details on network traffic and connectivity related to your Azure Functions. This includes data on inbound and outbound traffic, network latency, and potential network issues.
+
+Use this dashboard to:
+* Track network activity and performance of your Azure Functions.
+* Diagnose network-related issues and connectivity problems.
+* Monitor network usage and optimize network configuration.
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Azure-Functions/Azure-Functions-Network.png')} alt="Azure Functions Network dashboard" style={{border: '1px solid gray'}} width="800" />
```
Comment (JV0812) [inline docs/integrations/microsoft-azure/azure-functions.md:237]: Crop the extra space in the screenshot.

File: docs/integrations/microsoft-azure/azure-functions.md [hunk 22]
```diff
@@ -30,33 +41,276 @@ Azure service sends monitoring data to Azure Monitor, which can then [stream dat
...
+### Operations
+
+The **Azure Functions - Operations** dashboard provides an overview of operational metrics and activities related to your Azure Functions. This includes I/O Read, Write bytes/second and I/O Read, Write operations/second.
+
+Use this dashboard to:
+* Monitor the operational status and health of your Azure Functions.
+* Review deployment activities and execution metrics.
+* Analyze operational trends and optimize function management.
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Azure-Functions/Azure-Functions-Operations.png')} alt="Azure Functions Operations dashboard" style={{border: '1px solid gray'}} width="800" />
```
Comment (JV0812) [inline docs/integrations/microsoft-azure/azure-functions.md:248]: Crop the extra space in the dashboard image.

File: docs/integrations/microsoft-azure/azure-functions.md [hunk 23]
```diff
@@ -30,33 +41,276 @@ Azure service sends monitoring data to Azure Monitor, which can then [stream dat
...
+### Performance
+
+The **Azure Functions - Performance** dashboard provides insights into the performance of your Azure Functions. This includes metrics on execution times, throughput, and overall function efficiency.
+
+Use this dashboard to:
+* Monitor and analyze the performance of your Azure Functions.
+* Identify performance bottlenecks and optimize function execution.
+* Review performance metrics and trends to ensure optimal function performance.
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Azure-Functions/Azure-Functions-Performance.png')} alt="Azure Functions Performance dashboard" style={{border: '1px solid gray'}} width="800" />
```
Comment (JV0812) [inline docs/integrations/microsoft-azure/azure-functions.md:270]: Crop the extra space in the dashboard image.

File: docs/integrations/microsoft-azure/azure-functions.md [hunk 24]
```diff
@@ -30,33 +41,276 @@ Azure service sends monitoring data to Azure Monitor, which can then [stream dat
...
+
+### Health
+
+The **Azure Functions - Health** dashboard provides information of any service health incidents or resource health events associated with Azure Functions in your azure account.
+
+Use this dashboard to:
+* View recent resource and service health incidents.
+* View distribution of service and resource health by incident type.
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Azure-Functions/Azure-Functions-Health.png')} alt="Azure Functions health dashboard" style={{border: '1px solid gray'}} width="800" />
```
Comment (JV0812) [inline docs/integrations/microsoft-azure/azure-functions.md:280]: Please export the PNG file of the dashboard and add it to S3 bucket. The image added here looks like a normal screenshot, which actually got widened and the words are not clear in the preview page.

File: docs/integrations/microsoft-azure/azure-functions.md [hunk 25]
```diff
@@ -30,33 +41,276 @@ Azure service sends monitoring data to Azure Monitor, which can then [stream dat
...
+The **Azure Functions - Policy and Recommendations** dashboard provides information of all effect action operations performed by Azure policy and recommendations events from Azure Advisor.
+
+Use this dashboard to:
+* Monitor policy events with warnings and errors.
+* View recent failed policy events.
+* View total recommendation events.
+* Identify High Impact recommendations.
+* View recent recommendation events and navigate to the affected resource.
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Azure-Functions/Azure-Functions-Policy-and-Recommendations.png')} alt="Azure Functions - Policy and Recommendations dashboard" style={{border: '1px solid gray'}} width="800" />
```
Comment (JV0812) [inline docs/integrations/microsoft-azure/azure-functions.md:293]: Please export the PNG file of the dashboard and add it to S3 bucket. The image added here looks like a normal screenshot, which actually got widened and the words are not clear in the preview page.

File: docs/integrations/microsoft-azure/azure-functions.md [hunk 26]
```diff
@@ -30,33 +41,276 @@ Azure service sends monitoring data to Azure Monitor, which can then [stream dat
...
+### Administrative Operations
+
+The **Azure SQL - Administrative Operations** dashboard provides details on read/write/delete specific changes, different operations used, top 10 operations that caused most errors, and users performing admin operations.
+
+Use this dashboard to:
+* Identify top users performing administrative operations.
+* View Top 10 operations that caused the most errors.
+* View recent read, write, and delete operations.
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Azure-Functions/Azure-Functions-Administrative-Operations.png')} alt="Azure Functions Administrative Operations dashboard" style={{border: '1px solid gray'}} width="800" />
```
Comment (JV0812) [inline docs/integrations/microsoft-azure/azure-functions.md:304]: Please export the PNG file of the dashboard and add it to S3 bucket. The image added here looks like a normal screenshot, which actually got widened and the words are not clear in the preview page.

**Review by JV0812 — CHANGES_REQUESTED** [review]
Minor formatting and dashboard images fix.

File: docs/integrations/microsoft-azure/azure-functions.md (hunk 20 above)
Comment (JV0812) [inline docs/integrations/microsoft-azure/azure-functions.md:167]:
```suggestion
For getting health check metric, make sure you enable **Health check** under the **Monitoring** dropdown.
```

File: docs/integrations/microsoft-azure/azure-functions.md (hunk 21 above)
Comment (ankurch627) [reply docs/integrations/microsoft-azure/azure-functions.md:237]: Done

File: docs/integrations/microsoft-azure/azure-functions.md (hunk 22 above)
Comment (ankurch627) [reply docs/integrations/microsoft-azure/azure-functions.md:248]: Done

File: docs/integrations/microsoft-azure/azure-functions.md (hunk 23 above)
Comment (ankurch627) [reply docs/integrations/microsoft-azure/azure-functions.md:270]: Done

File: docs/integrations/microsoft-azure/azure-functions.md (hunk 25 above)
Comment (ankurch627) [reply docs/integrations/microsoft-azure/azure-functions.md:293]: Done

File: docs/integrations/microsoft-azure/azure-functions.md (hunk 26 above)
Comment (ankurch627) [reply docs/integrations/microsoft-azure/azure-functions.md:304]: Done

File: docs/integrations/microsoft-azure/azure-functions.md (hunk 24 above)
Comment (ankurch627) [reply docs/integrations/microsoft-azure/azure-functions.md:280]: Done

File: docs/integrations/microsoft-azure/azure-functions.md (hunk 19 above)
Comment (ankurch627) [reply docs/integrations/microsoft-azure/azure-functions.md:159]: Done

**Review by himanshu219 — APPROVED** [review]

**Review by JV0812 — APPROVED** [review]
LGTM

**Review by jpipkin1 — APPROVED** [review]
@ankurch627, I now see the he dashboard screenshots. Approved.

**Pull request merged**